src/components/JoinUs/Work.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import breakpoint from 'styled-components-breakpoint'
+import remcalc from 'remcalc'
+import { Link } from 'gatsby'
+import { Padding } from 'styled-components-spacing'
+import styled from 'styled-components'
+import { Row, CompensatedCol, CompensatedRow } from '../grid'
+import { H2, H3, Paragraph } from '../Typography'
+import colorLuminance from 'color-luminance'
+
+import {
+  Section,
+  FirstColumn,
+  SecondColumn,
+  Separator,
+  List,
+  TitleAndBody
+} from './elements'
+
+const MAX_CASE_STUDIES = 3
+
+/* Utility function to convert an HEX color into the RGB format */
+const hexToRgb = hex => {
+  var result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex)
+  return result
+    ? {
+        r: parseInt(result[1], 16),
+        g: parseInt(result[2], 16),
+        b: parseInt(result[3], 16)
+      }
+    : null
+}
+
+/* Returns the luminance (in a 0-255 range) of a color in the HEX format */
+const getColorLuminance = hexColor => {
+  const rgbValue = hexToRgb(`#${hexColor}`)
+
+  return colorLuminance(rgbValue.r, rgbValue.g, rgbValue.b)
+}
+
+const CaseStudyLink = styled(Link)`
+  background: ${props => props.bg};
+  color: ${props => props.color || props.theme.colors.white};
+  flex: 1;
+`
+
+const CaseStudy = ({ children, ...props }) => (
+  <CaseStudyLink {...props}>
+    <Padding
+      vertical={{ smallPhone: 2, smallTablet: 1.5, tablet: 3 }}
+      horizontal={{ smallPhone: 2, smallTablet: 1.5, tablet: 2 }}
+    >
+      {children}
```

Comment:
Since CaseStudy isn't being re-used anywhere, perhaps it's even clearer and more readable to pop the children (i.e. title and paragraph) directly in here?